docs/Uncommunicative-Method-Name.md

```diff
@@ -21,22 +21,24 @@ Reek's _Uncommunicative Method Name_ detector supports the
 
 | Option         | Value       | Effect  |
 | ---------------|-------------|---------|
-| `reject` | array of regular expressions or strings | The set of patterns / names that Reek uses to check for bad names. Defaults to `[/^[a-z]$/, /[0-9]$/, /[A-Z]/]`. |
-| `accept` | array of regular expressions or strings | The set of patterns / names that Reek will accept (and not report) even if they match one of the `reject` expressions. |
+| `reject` | array of strings | The set of names that Reek uses to check for bad names. Defaults to `[/^[a-z]$/, /[0-9]$/, /[A-Z]/]`. |
```

Comment:
This is now a little confusing since we say they're strings but the default is a bunch of regexes.

Reply:
Good point - what do you think about actually saying:

>> By default Reek rejects one-letter names, names ending in a number or names that have a capital letter in them

and then updating the other pieces of documentation that are similar to this in a similar fashion?